source/fundamentals/connection/connect.txt

```diff
@@ -97,11 +97,10 @@ verify that the connection is successful:
 
 .. note::
 
-   As of v4.7, the {+driver-short+} will automatically call the ``MongoClient.connect()``
-   method when using the client to interact with your MongoDB instance, **except** when
-   you use the client to call bulk operations. We recommend manually calling
-   ``MongoClient.connect()`` if you want granular control and insight over
-   your application's connections.
+   As of {+driver-short+} v4.7, the driver automatically calls the ``MongoClient.connect()``
```

Comment:
S: this is presumably getting backported to all active docs versions, and v4.7 is in legacy. Do we need to say when this behavior started?

```suggestion
   The {+driver-short+} automatically calls the ``MongoClient.connect()``
```

Reply:
Well spotted, happy to remove.